.php-cs-fixer.dist.php

```diff
@@ -10,9 +10,16 @@
             'imports_order' => ['class', 'function', 'const'],
         ],
         'declare_strict_types' => true,
+        'get_class_to_class_keyword' => false,
```

Comment:
`PHP Fatal error:  Cannot use ::class with dynamic class name` in PHP 7.4 and below